test/test_hid_transport.py

```diff
@@ -0,0 +1,85 @@
+import os
+import unittest
+from u2flib_host import hid_transport
+from u2flib_host import exc
+from u2flib_host.yubicommon.compat import byte2int, int2byte
+from mock import patch
```

Comment:
For some reason this seems to work on Travis, I'm not sure why exactly. It fails on my local machine when run under python 3, as the import there should be `unittest.mock`. Something like this should do the trick:
```
try:
    from unittest.mock import patch
except ImportError:  # On Python 2 use backport
    from mock import patch
```

Reply:
Sounds good